src/eureka/lib/citations.py

```diff
@@ -0,0 +1,529 @@
+CITATIONS = {
```

Comment:
We should also be citing Astraeus, jwst, crds, h5py, pandas, and xarray

For Astraeus, I did the following:
```
@misc{astraeus2022,
  author = {{Stevenson}, Kevin B.},
  title = "{{Astraeus}: A tool for standardizing I/O when reducing and analyzing exoplanet data}",
  year = {2022},
  publisher = {GitHub},
  journal = {GitHub repository},
  url = {https://github.com/kevin218/Astraeus}
}
```

For jwst, do the following:
```
@misc{jwst2022,
  author = {{JWST calibration pipeline developers}},
  title = "{{jwst}: Python library for science observations from the James Webb Space Telescope}",
  year = {2022},
  publisher = {GitHub},
  journal = {GitHub repository},
  url = {https://github.com/spacetelescope/jwst}
}
```

For CRDS, I did the following:
```
@misc{crds2022,
  author = {{CRDS developers}},
  title = "{{CRDS}: Calibration Reference Data System for HST and JWST}",
  year = {2022},
  publisher = {GitHub},
  journal = {GitHub repository},
  url = {https://github.com/spacetelescope/crds}
}
```

The others you should hopefully be able to find yourself